### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by degisew**
### Mobile Menu

This pull request is to add a mobile menu to a portfolio.

**In this pull request,  I did the following features:**

- The mobile menu appears when the user clicks (or taps) the hamburger button.
- The mobile menu disappears when the user clicks (or taps) the close (X) button.
- The mobile menu disappears when the user clicks (or taps) any of the mobile menu options.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,383 +1,490 @@
...
+            <a href="#">Portfolio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </div>
+          <button class="nav-menu" type="button">
+            <img
+              src="../images/Union.png"
+              alt="Menu drawer icon"
+              class="drawer-icon"
+            />
```
Comment (1GORDON) [inline index.html:36]: Great job team 🤝

- [ ] You may kindly fix this issue that occurs whenever I click on the portfolio option on the mobile menu. When I select the portfolio option, a different interface appears and it cannot allow me to click on the hamburger icon again.
- The interface that opens after selecting the portfolio option
profile two

- The correct interface that opens when contacts and about are clicked is different from the portfolio option.
first page

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @degisew & @eerapheal ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (degisew) [reply index.html:36]: Thank you. I will fix it.

**Review by mukhammadrizooff — APPROVED** [review]
Hi @degisew, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Menu working fine ✔️ 
- Used project requirements well ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**